### PR TITLE
Celery service setup

### PR DESCRIPTION
**Pull request by sc15zs**
Add Celery service to docker compose file for running background tasks. 

Ticket: https://dluhcdigital.atlassian.net/browse/FPASF-102

### CODE REVIEW

File: docker-compose.yml [hunk 1]
```diff
@@ -3,9 +3,9 @@ services:
     # https://github.com/communitiesuk/funding-service-design-post-award-data-store
     build:
       context: ../funding-service-design-post-award-data-store
-      args: 
+      args:
         REQUIREMENTS: requirements-dev.txt
-    command: bash -c "flask db upgrade && ${VSC_DEBUG:-flask run -p 8080 -h 0.0.0.0}"
+    command: bash -c "flask db upgrade && flask seed-ref && ${VSC_DEBUG:-flask run -p 8080 -h 0.0.0.0}"
```
Comment (samuelhwilliams) [inline docker-compose.yml:8]: Weird, I thought Marc had already added this 🤔

**Review by samuelhwilliams — APPROVED** [review]
Looks good to go once the data-store PR is merged 👍

File: docker-compose.yml [hunk 2]
```diff
@@ -22,11 +22,38 @@ services:
...
+    env_file: .env
+    environment:
+      - FLASK_ENV=development
+      - DATABASE_URL=postgresql://postgres:password@database:5432/data_store
+      - AWS_REGION=${AWS_REGION:-eu-central-1}
+      - AWS_ACCESS_KEY_ID=${AWS_ACCESS_KEY_ID:-test}
+      - AWS_SECRET_ACCESS_KEY=${AWS_SECRET_ACCESS_KEY:-test}
+      - AWS_ENDPOINT_OVERRIDE=http://localstack:4566/
+      - AWS_S3_BUCKET_FAILED_FILES=data-store-failed-files-dev
+      - AWS_S3_BUCKET_FIND_DATA_FILES=data-store-find-data
```
Comment (gidsg) [inline docker-compose.yml:50]: Does the worker actually need these env vars? Does it start up without them? Would be good to get the env vars down to a minimal set.

Comment (sc15zs) [reply docker-compose.yml]: Removed these from celery setup.

**Pull request merged**